pipelines/android.yml

```diff
@@ -0,0 +1,22 @@
+# Build for Android APK
+
+variables:
+- template: config/settings.yml
+
+jobs:
+- job: AndroidValidation
+  timeoutInMinutes: 60
+  pool:
+    name: On-Prem Unity
+    demands:
+    - Unity2019.4.0f1
+    - COG-UnityCache-WUS2-01
+    - SDK_18362 -equals TRUE
+  steps:
+  - template: templates/leap-tests.yml
```

Comment:
I might be looking at the wrong place but I can't see this template on the target branch (feature/android_validation).

Reply:
I'm trying to add it to the target branch is the thing... trying to add it directly didn't work so I opened a PR for it